app/services/deprecation_warnings/disallowed_deprecations.rb

```diff
@@ -11,9 +11,17 @@ class ::DisallowedDeprecationError < StandardError; end
     /Dangerous query method \(method whose arguments are used as raw SQL\) called with non\-attribute argument\(s\)/
   ].freeze
 
+   # Regular expressions for Rails 6.1 deprecation warnings that we have addressed in the codebase
+  RAILS_6_1_FIXED_DEPRECATION_WARNING_REGEXES = [
+    /update_attributes!*\(/,
+    /render file: should be given the absolute path to a file/,
+    /ActionView::Base instances should be constructed with a lookup context, assignments, and a controller./
+  ].freeze
```

Comment:
Let's leave out `/update_attributes!*\(/` here, since it is unrelated to the deprecations being fixed in this story story.

```suggestion
   # Regular expressions for Rails 6.1 deprecation warnings that we have addressed in the codebase
  RAILS_6_1_FIXED_DEPRECATION_WARNING_REGEXES = [
    /render file: should be given the absolute path to a file/,
    /ActionView::Base instances should be constructed with a lookup context, assignments, and a controller./
  ].freeze
```